packages/vscode-extension/package.json

```diff
@@ -62,8 +72,9 @@
 		"@types/vscode": "^1.40.0",
 		"glob": "^7.1.5",
 		"mocha": "^6.2.2",
-		"typescript": "^3.6.4",
 		"tslint": "^5.20.0",
+		"typescript": "^3.6.4",
+		"vsce": "^1.75.0",
```

Comment:
Do we really need vsce as a dependency of  the vscode-extension?
Won't we be it installed globally?

Reply:
I'll remove it